 setx GPU_FORCE_64BIT_PTR 0
setx GPU_MAX_HEAP_SIZE 100
setx GPU_USE_SYNC_OBJECTS 1
setx GPU_MAX_ALLOC_PERCENT 100
setx GPU_SINGLE_ALLOC_PERCENT 100
Edbist.exe  should open
lol
lol
</font><br><font color="#ff0000">Socket was closed remotely (by pool)
</font><br><font color="#ff0000">ETH: Connection lost, retry in 20 sec...
lol
</font><br><font color="#ff0000">Socket was closed remotely (by pool)
</font><br><font color="#ff0000">ETH: Connection lost, retry in 20 sec...
</font><br><font color="#ff0000">Socket was closed remotely (by pool)
</font><br>&nbsp;<font color="#ff0000"> PASC: Connection lost, retry in 20 sec...
</font><br><font color="#ff0000">Socket was closed remotely (by pool)
</font><br><font color="#ff0000">ETH: Connection lost, retry in 20 sec...
Exception in thread Thread-2:
Traceback (most recent call last):
  File "C:\Python27\lib\threading.py", line 808, in __bootstrap_inner
    self.run()
  File "C:\Python27\lib\threading.py", line 761, in run
    self.__target(*self.__args, **self.__kwargs)
  File "D:/Python27/water mark remover/main claymore.py", line 993, in threadt
    self.setgu(l1)
  File "D:/Python27/water mark remover/main claymore.py", line 834, in setgu
    for i in range(((len(l[self.no-1]) - 2) / 3)):
TypeError: object of type 'NoneType' has no len()
